airflow/models/taskinstancehistory.py

```diff
@@ -83,6 +84,8 @@ class TaskInstanceHistory(Base):
     executor_config = Column(ExecutorConfigType(pickler=dill))
     updated_at = Column(UtcDateTime, default=timezone.utcnow, onupdate=timezone.utcnow)
     rendered_map_index = Column(String(250))
+    context_carrier = Column(MutableDict.as_mutable(ExtendedJSON))
+    span_status = Column(String(50), default=SpanStatus.NOT_STARTED)
```

Comment:
We don't need these on TIHistory -- rows only make it here once they are finished/done, so they will never have a span about them.

Reply:
There is a pre-commit error if we don't add them

```
Check that TI and TIS have the same attributes..............................Failed
- hook id: check-taskinstance-tis-attrs
- exit code: 1

Attributes in TaskInstance but not in TaskInstanceHistory: 

 {'context_carrier', 'span_status'}

 Please make sure that the attributes are consistent across the classes.
```